pkg/migrator/migrator.go

```diff
@@ -252,8 +252,11 @@ func IsPackageInOpsrc(packageName string, opsrc *v1.OperatorSource) bool {
 // extracts `community` off the name, appends `operators`
 // to it, and returns `community-operators` as output
 func ExtractCsName(cscName string) string {
-	possibleCsName := strings.Split(cscName, "-")[1]
-	return fmt.Sprintf("%s-%s", possibleCsName, "operators")
+	possibleCsName := strings.Split(cscName, "-")
+	if len(possibleCsName) > 1 {
+		return fmt.Sprintf("%s-%s", possibleCsName[1], "operators")
+	}
+	return return fmt.Sprintf("%s-%s", possibleCsName[0], "operators")
```

Comment:
does this work in every case? should we have some more fundamental base case here instead?

Reply:
Maybe we can open a separate PR to improve the implementation of this function?